WooCommerce/WooCommerceTests/ViewRelated/Orders/Issue Refund/IssueRefundViewModelTests.swift

```diff
@@ -342,4 +342,52 @@ final class IssueRefundViewModelTests: XCTestCase {
         // Price is 11.50 and tax is 0.99 (2.97 / 3(quantity))
         XCTAssertEqual(viewModel.title, "$12.49")
     }
+
+    func test_viewModel_starts_with_next_button_disabled() {
+        // Given
+        let currencySettings = CurrencySettings()
```

Comment:
I noticed that the three tests start with the same arrange section. 

I am curious here, are we trying to avoid extracting shared code to private methods in the tests, or do we consider that a bit of repetition in the individual tests actually makes them more readable, and therefore it's a good thing?

Reply:
I like to have unit tests as isolated as possible so they make sense and exists on their own. I guess this is something we have not yet discussed but I'm happy to hear alternatives to improve them!